integration/301_internet_edge_with_ebpf_test.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
+
+# shellcheck disable=SC1091
+. ./config.sh
+
+start_suite "Test short lived connections from the Internet"
+
+if ! echo "$HOST1" | grep "us-central1-a"; then
+    echo "Skipping; test needs to be run against VMs on GCE."
+    scope_end_suite
+    exit
+fi
+
+weave_on "$HOST1" launch
+scope_on "$HOST1" launch --probe.ebpf.connections=true
+docker_on "$HOST1" run -d -p 80:80 --name nginx nginx
+
+do_connections() {
+    while true; do
+        curl -s "http://$HOST1:80/" >/dev/null || true
+        sleep 1
+    done
+}
+do_connections &
+
+wait_for_containers "$HOST1" 60 nginx "The Internet"
+
+has_connection_by_id containers "$HOST1" "in-theinternet" "$(node_id containers "$HOST1" nginx)"
```

Comment:
Isn't there a race condition here? Isn't it possible that the connection hasn't shown up in the app by the time this is run?